tests/scripts/arm32_ci_script.sh

```diff
@@ -102,7 +102,15 @@ function check_git_head {
 function unmount_rootfs {
     local rootfsFolder="$1"
 
-    if grep -qs "$rootfsFolder" /proc/mounts; then
+    #Check if there are any open files in this directory.
+    if [ -d $rootfsFolder ]; then
+        #If we find information about the file
+        if [ sudo lsof +D $rootfsFolder ]; then
```

Comment:
Bash is not my forté, this if is here just to make sure that when information is found the build does not crash. If there is a more elegant way to address this please let me know.